src/main/java/seedu/address/model/util/SampleDataUtil.java

```diff
@@ -8,37 +8,60 @@
 import seedu.address.model.ReadOnlyAddressBook;
 import seedu.address.model.person.Email;
 import seedu.address.model.person.Name;
-import seedu.address.model.person.Person;
 import seedu.address.model.person.Phone;
 import seedu.address.model.person.doctor.Doctor;
+import seedu.address.model.person.doctor.Specialty;
+import seedu.address.model.person.doctor.Yoe;
+import seedu.address.model.person.patient.Diagnosis;
+import seedu.address.model.person.patient.Height;
 import seedu.address.model.person.patient.Patient;
+import seedu.address.model.person.patient.Remark;
+import seedu.address.model.person.patient.Status;
+import seedu.address.model.person.patient.Weight;
 import seedu.address.model.tag.Tag;
 
 /**
  * Contains utility methods for populating {@code AddressBook} with sample data.
  */
 public class SampleDataUtil {
-    public static Person[] getSamplePersons() {
-        return new Person[] {
-            new Person(new Name("Alex Yeoh"), new Phone("87438807"), new Email("alexyeoh@example.com"),
-                getTagSet("friends")),
-            new Person(new Name("Bernice Yu"), new Phone("99272758"), new Email("berniceyu@example.com"),
-                getTagSet("colleagues", "friends")),
-            new Person(new Name("Charlotte Oliveiro"), new Phone("93210283"), new Email("charlotte@example.com"),
-                getTagSet("neighbours")),
-            new Person(new Name("David Li"), new Phone("91031282"), new Email("lidavid@example.com"),
-                getTagSet("family")),
-            new Person(new Name("Irfan Ibrahim"), new Phone("92492021"), new Email("irfan@example.com"),
-                getTagSet("classmates")),
-            new Person(new Name("Roy Balakrishnan"), new Phone("92624417"), new Email("royb@example.com"),
-                getTagSet("colleagues"))
+
+    public static Doctor[] getSampleDoctors() {
```

Comment:
I believe sampleDoctors and sampleAddressBook is not used. Do we want to remove it completely?

Reply:
Removed post discussion